hbase-agent/src/main/java/org/apache/ranger/authorization/hbase/RangerAuthorizationCoprocessor.java

```diff
@@ -491,14 +515,52 @@ ColumnFamilyAccessResult evaluateAccess(ObserverContext<?> ctx, String operation
 				session.resourceMatchingScope(RangerAccessRequest.ResourceMatchingScope.SELF);
 				session.ignoreDescendantDeny(true);
 			} else {
-				LOG.debug("evaluateAccess: columns collection not empty.  Skipping Family level check, will do finer level access check.");
+				boolean isColumnAuthShortCircuitingEnabled = hbasePlugin.getPropertyIsColumnAuthOptimizationEnabled();
```

Comment:
To be consistent, rename: `isColumnAuthShortCircuitingEnabled` => `isColumnAuthOptimizationEnabled`